superset-frontend/src/dashboard/components/RefreshIntervalModal.tsx

```diff
@@ -140,16 +256,52 @@ class RefreshIntervalModal extends React.PureComponent<
         }
         modalFooter={
           <>
+            <Button onClick={this.onCancel} buttonSize="small">
+              {t('Cancel')}
+            </Button>
             <Button
               buttonStyle="primary"
               buttonSize="small"
-              onClick={this.onSave}
+              onClick={() => {
```

Comment:
It would be better if handlers called an external function rather than defining the logic inline.

Reply:
Done